components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -474,7 +481,13 @@ public enum ErrorMessages {
         ERROR_CODE_ERROR_FIRING_EVENTS("65090", "Error while firing the events",
                 "Server encountered an error while firing the events"),
         ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_DEPTH("65091", "Unable to retrieve the organization depth.",
-                "Server encountered an error while retrieving depth of organization with ID: %s.");
+                "Server encountered an error while retrieving depth of organization with ID: %s."),
+        ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_APPLICATIONS("65092", "Error while retrieving applications.",
+                "Encountered an error while retrieving applications in organization id %s."),
+        ERROR_CODE_ERROR_CHECKING_APPLICATION_IS_A_FRAGMENT("65093", "Error while checking whether fragment app.",
```

Comment:
`Error while checking whether fragment app` msg is not clear. Shall we have something like `Error while checking whether app is a fragment`

Reply:
resolved with 2faf5c2e64b44d7a3e1145676bd4418471d360c1